includes/core/classes/class-setup.php

```diff
@@ -111,6 +113,38 @@ protected function setup_hooks(): void {
 		);
 	}
 
+	/**
+	 * Load plugin textdomain.
+	 * 
+	 * Calling load_plugin_textdomain() should be delayed until init action.
+	 * @see https://developer.wordpress.org/reference/functions/load_plugin_textdomain/#comment-1568
+	 *
+	 * load_plugin_textdomain() will try to load the mo file firstly from:
+	 * WP_LANG_DIR . ‘/plugins/’ . $mofile
+	 * Only if it is not able to it will load it from GATHERPRESS_DIR_NAME folder
+	 * @see https://developer.wordpress.org/reference/functions/load_plugin_textdomain/#comment-3521
+	 *
+	 * Since WordPress 4.6 translations now take translate.wordpress.org as priority and so
+	 * plugins that are translated via translate.wordpress.org DO NOT necessary require load_plugin_textdomain() anymore.
+	 * If you don’t want to add a load_plugin_textdomain() call to your plugin
+	 * you have to set the Requires at least: field in your readme.txt to 4.6 or more.
+	 * @see https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#plugins-on-wordpress-org
+	 *
+	 * BUT, ...!!!
+	 *
+	 * A call to load_plugin_textdomain() is ABSOLUTELY NEEDED to properly load all translation files
+	 * for users how chosed another than the sites default language in their /profile.php.
+	 */
+	public function load_gatherpress_textdomain() {
+		// Keep as a fallback if translation files were not loaded from translate.wordpress.org ??
+		// load_plugin_textdomain( 'gatherpress', false, GATHERPRESS_DIR_NAME . '/languages' );
+	
+		// Or use it like that
+		// and DELETE the /languages folder,
+		// and "* Domain Path:       /languages" as well?
+		load_plugin_textdomain( 'gatherpress', false, false ); 
```

Comment:
Should we keep the `/languages` folder or remove it entirely @mauteri ?

Reply:
it should have been removed entirely since we are on .org repo now and no longer require it. thx!